spec/services/feeds/import_spec.rb

```diff
@@ -12,7 +12,7 @@
       end
     end
 
-    it "fetch only articles from a feed_url", vcr: { cassette_name: "feeds_import" } do
+    xit "fetch only articles from a feed_url", vcr: { cassette_name: "feeds_import" } do
```

Comment:
I could use help with the tests in this file! I can see that the feeds importer service is returning a number, but I can't quite figure out _what_ exactly we're testing in these specs that I've skipped here.

Maybe @rhymes or @maestromac have some suggestions on how to convert these away from using the `approvals` gem's `verify` (which won't work anymore), into something that is actually useful here? 🤔

Reply:
I think we're just testing that `num_articles` is 21, that's it :D To make sure it's always importing the same amount of `<item>`s in the RSS feed (you could either hardcode the number or, probably better, XML-parse the file and check that `num_articles` corresponds to the number of `<item>` tags, maybe?)

This was borrowed by an older `RssReader` spec